dpnp/random/_random.pyx

```diff
@@ -111,6 +113,33 @@ cpdef dparray dpnp_exponential(double beta, size):
     return result
 
 
+cpdef dparray dpnp_gamma(double shape, double scale, size):
+    """
+    Return a random matrix with data from the "gamma" distribution.
```

Comment:
Why is it call "random matrix" ? The code populates a flat buffer with random numbers. 

"Random matrix" has special meaning in computation math, and this has nothing to do with it.

Returns an array populated with samples from gamma distribution. 

Quotes around gamma are not needed.

Reply:
Thank you. This comment is common for all other functions. I will refactor on other PR

Reply:
Fix for all rest functions on #186